### DISCUSSION TITLE
Wallet Hacked ( need Help please)

### DISCUSSION THREAD

**Discussion by sanook999** [blockscout/blockscout#9715] [Q&A]
Dear all,
My metamask wallet on the Manta Pacific chain has been drained !
It happens one week ago ...
All my STONE has been send to unknown address !
Is there any hope to get my STONE back?
details of transaction :0x2ec7a2d1d12f8a949a9ddcb741646ebbc2330b7f78b954d6ef85fdb133ec0053
Can I contact the Manta team so they can solve this ?
Many thanks for all help.

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Comment by sanook999** [comment]
Can't find chain support team..On 25 Mar 2024, at 18:37, Maxim Filonov ***@***.***> wrote:﻿
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Reply by sl1depengwyn** [reply]
https://discord.com/invite/mantanetwork

**Comment by hakouchrayet-oss** [comment]
Bonjour

**Comment by hakouchrayet-oss** [comment]
Hi

**Comment by robertscfurey-ops** [comment]
> Dear all,
> 
> My metamask wallet on the Manta Pacific chain has been drained !
> 
> It happens one week ago ...
> 
> All my STONE has been send to unknown address !
> 
> Is there any hope to get my STONE back?
> 
> details of transaction :0x2ec7a2d1d12f8a949a9ddcb741646ebbc2330b7f78b954d6ef85fdb133ec0053
> 
> Can I contact the Manta team so they can solve this ?
> 
> Many thanks for all help.

Have you been able to recover your stolen funds?? Reach out to help.coinbase.io117@gmail.com
